services/storage/object/metadata.yaml

```diff
@@ -0,0 +1,15 @@
+title: CCC Object Storage Security Threats
+category-id: CCC.ObjStor
+
+# Assurance Levels:
+## None if actively under development
+## AL0 if only features are complete at release time
+## AL1 if features, threats, and controls are complete at release time
+## AL2 if threats are written based on a threat model for this category
+## AL3 if threats are written based on a red team exercise for this category
+assurance_level: None
+
+threat_model_author: None
+threat_model_url: None
+red_team: None
+red_team_exercize_url: None
```

Comment:
@mlysaght2017 please put a close eye on this... I think it might diverge a bit from what we discussed on previous calls. 

My thought is that this allows us to specify the highest assurance level that a resource can be certified to with the current state of the resource.

Also, I'm imagining that setting `None` is a good way to make sure that no assets are accidentally released with known issues.

Reply:
Agreed! I'll open a PR with a rephrase to the assurance levels guideline.